### DISCUSSION TITLE
Welcome to ImageGoNord Discussions!

### DISCUSSION THREAD

**Discussion by TheJoin95** [Schrodinger-Hat/ImageGoNord#48] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by paulopacitti** [comment]
I'm such a big fan of **ImageGoNord** that I made [gruvbox-factory](https://github.com/paulopacitti/gruvbox-factory) 🏭 Built using **ImageGoNord** lib, it's possible to manufacture gruvbox themed wallpapers. @Wabri is building a new `cli` for **IGN** and invited me to help this community and make **IGN** bigger ⭐
